### PR TITLE
Basic calculator working

### PR DESCRIPTION
**Pull request by IvanG16**

### CODE REVIEW

**Review by mfigares — COMMENTED** [review]
I n order to evaluate this PR properly, you should get the node_modules folder in the gitignore. If not, we have this massive file count that slows down even the github PR interphase, and it shouldn't be necessary to keep them in the repo

File: src/components/button.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <button onClick={onClick} type="button" className={backgroundColor}>
+      <div className="w-1/2 text-black text-2xl font-normal Inconsolata">
+        {value}
+      </div>
+    </button>
+  );
+}
+
+Button.propTypes = {
+  buttonClick: PropTypes.func.isRequired,
```
Comment (naipeloz) [inline src/components/button.jsx:22]: The name must be more descriptive. Something like handleClick or onClick

File: src/components/button.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Button({ value, buttonClick, valueType }) {
+  const onClick = () => {
+    buttonClick(value);
+  };
+
+  const styles = 'w-[81px] h-[45px] p-2.5 rounded-[5px] border border-black flex-col justify-center items-center gap-2.5 inline-flex';
+  const backgroundColor = valueType === 'operand' ? styles.concat(' bg-stone-50') : styles.concat(' bg-stone-300');
```
Comment (naipeloz) [inline src/components/button.jsx:10]: It is ok. But, you could simplify the style definition doing something like this:
```
const styles = `w-[81px] h-[45px] p-2.5 rounded-[5px] border border-black flex-col justify-center items-center gap-2.5 inline-flex ${valueType === 'operand' ? 'bg-stone-50' : 'bg-stone-300'} `
```

File: package-lock.json
Comment (naipeloz) [inline package-lock.json:1]: It is not mandatory. But, it could be ignored by git

File: README.md
Comment (naipeloz) [inline README.md:1]: Update the README. with a nice description of your practice

**Review by naipeloz — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (mfigares) [reply package-lock.json:1]: I think I'd leave it, that way we have the actual _full_ dependency tree for the project, don't you think? 
Most of the time, it doesn't matter but sometimes, having a package-lock.json lets you identify and fix dependency issues ()

**Review by mfigares — APPROVED** [review]
Almost legally approved! 🚀

**Pull request merged**